src/util/child_common.c

```diff
@@ -776,6 +769,13 @@ static errno_t prepare_child_argv(TALLOC_CTX *mem_ctx,
             goto fail;
         }
 
+        argv[--argc] = talloc_asprintf(argv, "--backtrace=%d",
+                                       sss_get_debug_backtrace_enable());
```

Comment:
Hi,

since you are using `?:` with `sss_set_debug_backtrace_enable()` to translate `int` to `bool` I wonder if you would like to use `?:` here as well to translate `bool` to `int`. If I understand implicit type conversion correctly it is safe to skip the translation in both cases. Personally I'd tend use the `?:` translation but I'm fine either way.

bye,
Sumit

Reply:
Fixed.

Reply:
Thanks.